apps/app-article/src/App.js

```diff
@@ -77,11 +89,19 @@ class App extends Component {
         if (getUrlParam().has('login')) {
             this.setState({ forceLoginView: true });
         } else {
-            this.getArticle();
-            this.getMostReadArticles();
+            this.getArticle(),
```

Comment:
Thought this was a syntax error but apparently it's not 😄 Why this change though?

Reply:
This should NOT be there!